### ISSUE TITLE
Should the cursor blink for text input?

### ISSUE THREAD

**Issue by Xaeroxe** [iced-rs/iced#89]
Making the text entry cursor blink about twice a second would require re-drawing the frame periodically, but would align more with typical user expectations.

**Comment by hecrj** [comment]
Yes! This is planned once #31 lands.

**Comment by hecrj** [comment]
Closing in favor of #560.
